Remove PHPCS workflow

greg0ire opened the pull request:
It shouldn't have been introduced in the first place, as there were no files to check inside `lib` anymore.

greg0ire committed:
Remove PHPCS workflow

It should not have been introduced, as there were no PHP files to check
inside lib.

simPod reviewed (changes requested):
We can keep it as a smoke test as done in #364.

Or remove the whole job, not just the step?

greg0ire committed:
Remove unneeded file tag

We only every call phpcbf with an argument, so this tag never gets used.
It was useful before 69af948fd88279c198f6361d37088603ee508bb9, when
there were sniffs inside lib.

greg0ire commented:
I've seen the smoke test and wasn't convinced it was necessary. Good point about removing the whole job though, I definitely missed that!
